readme/acsf-setup.md

```diff
@@ -3,9 +3,11 @@
 To configure a project to run on ACSF, perform the following steps after initially setting up BLT:
 
 1. Execute `blt acsf:init` from the project root.
-1. Create a custom module in `docroot/modules/custom` that will be used in lieu of a custom profile.
-1. Add `acsf` as a dependency for your custom module.
-1. If using lightning, create a custom [lightning.extend.yml](https://github.com/acquia/lightning/blob/8.x-1.x/lightning.extend.yml) and add it to `sites/default`
+1. Create a custom subprofile as described [here](https://docs.acquia.com/lightning/subprofile).
```

Comment:
I think we need to cover both Lightning and non-Lightning use cases. For Lightning, use a sub profile. Otherwise create a new custom profile, which can easily be done with Drupal console.